src/sections/Home/So-Special-Section/so-special-style.js

```diff
@@ -39,6 +61,29 @@ const SoSpecialWrapper = styled.div`
             height:100%;
         }    
     }
+    .slick-prev { 
+        left: -2.5rem;
+        position: relative;
+        top: 14.90rem;
```

Comment:
Not needed

Reply:
I had put this to keep both the arrows at the same level.

Reply:
Once you set the line-height, it will be at the same height